hybridse/src/vm/transform.cc

```diff
@@ -887,6 +891,17 @@ Status BatchModeTransformer::TransformQueryPlan(const ::hybridse::node::PlanNode
     return TransformPlanOp(node->GetChildren()[0], output);
 }
 
+Status BatchModeTransformer::TransformLoadDataOp(const node::LoadDataPlanNode* node,
+                                   PhysicalOpNode** output){
+    CHECK_TRUE(node != nullptr && output != nullptr, kPlanError,
```

Comment:
ASSERT RequestMode unsupport

Reply:
ASSERT unsupport in runner.cc

Reply:
request transformer assert done
runner won't support LoadDataOp, so stay the same.